### PR TITLE
Migrate '/channel_layout/center_channel' module to TypeScript

### PR DESCRIPTION
**Pull request by bolariin**
#### Summary

This PR migrates 'components/channel_layout/center_channel' module and associated tests to TypeScript

#### Ticket Link
Resolves https://github.com/mattermost/mattermost-server/issues/13733

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@formatjs/intl-pluralrules": "1.3.9",
     "@formatjs/intl-relativetimeformat": "4.5.1",
+    "@types/classnames": "^2.2.6",
```
Comment (saturninoabril) [inline package.json]: Looks like this is not needed.  Could you please confirm?

Comment (devinbinnie) [reply package.json]: It is needed, we're using the `classNames` library in `center_channel.tsx`

Comment (saturninoabril) [reply package.json]: Thanks @devinbinnie.  Then, my only request is to have it on fix version
```suggestion
    "@types/classnames": "2.2.6",
```

Comment (devinbinnie) [reply package.json]: Agreed.

**Comment by hmhealey** [comment]
Thanks for the PR and for helping with this migration!

**Pull request merged**